testing/runfossa/runfossa.go

```diff
@@ -3,18 +3,42 @@ package runfossa
 import "github.com/fossas/fossa-cli/exec"
 
 // Init executes fossa init the provided directory
-func Init(projectDir string) error {
+func Init(projectDir string) (string, error) {
+	output, err := runfossa(projectDir, []string{"init"})
```

Comment:
Why is this command name hardcoded when others are not? Let's hardcode either all or none of them. (My preference is for all of them, especially since the commands are not referenced anywhere else.)